sandbox-backend/src/main/resources/db/migration/V0000001__addLocation.sql

```diff
@@ -1,6 +1,6 @@
-INSERT INTO location (city) VALUES ('Lviv');
-INSERT INTO location (city) VALUES ('Kyiv');
-INSERT INTO location (city) VALUES ('Kharkiv');
-INSERT INTO location (city) VALUES ('Minsk');
-INSERT INTO location (city) VALUES ('Grodno');
-INSERT INTO location (city) VALUES ('Brest');
\ No newline at end of file
+INSERT INTO location (city, country) VALUES ('Lviv', 'Ukraine');
```

Comment:
There was broken The Third normal form. country depends on city but not on identifier. Usually it is better to have tree hierarchy for Location table. Anyway it is not critical not but we have to discuss that on the next meeting.

Reply:
okay